training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Use an approved example bucket name, <code>DOC-EXAMPLE-BUCKET</code>	, rather than "your-bucket".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>It seems the description is not perfectly matching with the following code. Updating the paragraphs.</p>